docs/guides/socket-collector.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'The "socket-collector" gadget'
+weight: 10
+---
+
+socket-collector gathers information about TCP and UDP sockets. The "LOCAL" is
```

Comment:
This LOCAL is lacking a noun. column? item?
Same with REMOTE in the next line.
Alternatively, I think that if you just remove the article it would be fine (like what you did with STATUS in the last line)

Reply:
Changes applied.